modules/uhttp/http.go

```diff
@@ -79,7 +79,7 @@ type Module struct {
 	listener net.Listener
 	handlers []RouteHandler
 	listenMu sync.RWMutex
-	filters  []Filter
+	fcb      FilterChainBuilder
```

Comment:
The call wrapping happens in Start, which is where filter chain is getting built.